src/components/buyers/list/BuyerList.tsx

```diff
@@ -0,0 +1,232 @@
+          User Groups ({buyersMeta[row.original.ID]["userGroupsCount"]})
+        </Button>
+        <Button onClick={() => router.push(`/buyers/${row.original.ID}/users`)} variant="outline">
+          Users ({buyersMeta[row.original.ID]["usersCount"]})
+        </Button>
+      </ButtonGroup>
+    )
+  }
+
+  const CatalogColumn: DataTableColumn<IBuyer> = {
+    header: "CATALOGS",
+    cell: ({row}) => (
+      <Link href={`/buyers/${row.original.ID}/catalogs`}>
+        <Button variant="outline">Catalogs ({buyersMeta[row.original.ID]["catalogsCount"]})</Button>
+      </Link>
+    )
+  }
+
+  const ActionsColumn: DataTableColumn<IBuyer> = {
+    header: "ACTIONS",
+    cell: ({row}) => (
+      <ButtonGroup>
+        <Button variant="outline" onClick={() => router.push(`/buyers/${row.original.ID}/`)}>
+          Edit
+        </Button>
+        <Button variant="outline" onClick={() => deleteBuyer(row.original.ID)}>
+          Delete
+        </Button>
+      </ButtonGroup>
+    )
+  }
+
+  const BuyerTableOptions: ListViewTableOptions<IBuyer> = {
+    responsive: {
+      base: [IdColumn, NameColumn],
+      md: [IdColumn, NameColumn],
+      lg: [IdColumn, NameColumn],
+      xl: [
+        IdColumn,
+        NameColumn,
+        DefaultCatalogIDColumn,
+        StatusColumn,
+        CreatedDateColumn,
+        UserGroupColumn,
+        CatalogColumn,
+        ActionsColumn
+      ]
+    }
+  }
+
+  return (
+    showList && (
+      <ListView<IBuyer>
+        service={Buyers.List}
+        tableOptions={BuyerTableOptions}
+        paramMap={paramMap}
+        queryMap={BuyerQueryMap}
+      >
+        {({renderContent, meta, updateQuery, queryParams, viewModeToggle, ...listViewChildProps}) => (
+          <Container maxW="100%">
+            <Box>
+              <Stack direction="row" mb={5}>
+                <Stack direction={["column", "column", "column", "row"]}>
+                  <DebouncedSearchInput
+                    label="Search buyers"
+                    value={queryParams["Search"]}
+                    onSearch={updateQuery("s", true)}
+                  />
+                </Stack>
+                <Box as="span" flexGrow="1"></Box>
+                <Stack direction={["column", "column", "column", "row"]}>
+                  <Stack direction="row" order={[1, 1, 1, 0]}>
+                    {meta && <ListViewMetaInfo range={meta.ItemRange} total={meta.TotalCount} />}
+                    <Box as="span" width="2"></Box>
+                    {viewModeToggle}
+                  </Stack>
+                  <Box order={[0, 0, 0, 1]} mt={0}>
+                    <Link passHref href="/buyers/add">
+                      <Button variant="solid" colorScheme="primary" as="a" mb={3}>
+                        Create Buyer
+                      </Button>
+                    </Link>
+                  </Box>
+                </Stack>
+              </Stack>
+            </Box>
```

Comment:
So while it's not absolutely necessary to create a separate BuyerListToolbar component, let's try and follow a similar file structure for our default examples. See how @maxwmaher splits out his components in admin settings.